site/_scss/blocks/_event-card.scss

```diff
@@ -0,0 +1,200 @@
+.event-card {
+  background: get-color('grey-50');
+
+  &[show-details='show-details'] {
```

Comment:
Still not boolean?

Reply:
I think this is the standard for boolean attributes (as defined here https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#boolean-attributes). Can switch out to use 'true' for true and '' for false though if that's the preference?

Reply:
I think the meaning is that 'false' does not resolve to False in attributes, but is used as any other string. The boolean way would be to add and remove show-details attribute (without value) rather than changing it's string value. But it's ok either way, it's just a nit - can stay as is.